tests/check_framework/models.py

```diff
@@ -6,10 +6,14 @@ class SimpleModel(models.Model):
     field = models.IntegerField()
     manager = models.manager.Manager()
 
+class CheckModel(models.Model):
+    check_ran = models.BooleanField(default=False)
+
 
 @register('tests')
 def my_check(app_configs, **kwargs):
     my_check.did_run = True
+    CheckModel.objects.create(check_ran=True).save()
```

Comment:
```suggestion
    CheckModel.objects.create(check_ran=True)
```

No need to call `save()` - `create()` already saves under the hood.

Reply:
Good catch!